app/components/ui/Footer.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Link } from 'react-router';
 import config from '../../config';
 
-const subDomain = window.location.host.split('.')[0];
+const domain = window.location.host.split('.')[0];
```

Comment:
This logic won't work for grabbing the root domain right?

Reply:
it does since we are passing in `sfserviceguide` rather than `sfserviceguide.org` as our MOCHD_DOMAIN value. i think it's arguably preferable over say just checking `window.location.host` since it will also work for `staging.sfserviceguide.org`

Reply:
Does this work for `staging.sfserviceguide.org` though? Won't that code find `staging` ?

Reply:
I think we update it to 
```
window.location.host.split('.')[1]
```
We would be ok

Reply:
ack you are right about that; however if we run `window.location.host.split('.')[1]` against `https://sfserviceguide.org` (without the prefixed www), it returns `.org`

i think the most robust/flexible solution, short of using a parsing library (i don't think we'd be able to get that in in time for this deploy), would be to check the window.location.host for a substring `sfserviceguide`– e.g. `window.location.host.indexOf(config.MOHCD_DOMAIN) > -1` 

i've tested that code against staging.sfserviceguide.org, http://sfserviceguide.org, and www.sfsergiveguide.org and it works for all three. 

i will update my PR